src/ShoppingCart/shoppingCart.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { BreadCrumb } from '../MultipurposeComponents/BreadCrumb/breadCrumb';
+import { CartGoods } from '../Additional/additional_api';
+import { CatalogMaker } from '../MultipurposeComponents/CatalogMaker/catalogMaker';
+
+export const ShoppingCart = () => {
+  return (
+    <div className="Favourites">
+      <BreadCrumb page="Shopping cart" />
+      <h1 className="Favourites__title">Shopping cart</h1>
+      {
+        CartGoods.length !== 1 ? (
```

Comment:
why this var with uppercase first letter?

Reply:
CartGoods and FavGoods were renamed to cartGoods & favGoods